jans-cedarling/cedarling/examples/authorize_without_jwt_validation.rs

```diff
@@ -0,0 +1,135 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use cedarling::{
+    BootstrapConfig, Cedarling, JwtConfig, LogConfig, LogTypeConfig, PolicyStoreConfig,
+    PolicyStoreSource, Request, ResourceData,
+};
+use std::collections::HashMap;
+
+// The human-readable policy and schema file is located in next folder:
+// `test_files\policy-store_ok`
+static POLICY_STORE_RAW: &str = include_str!("../../test_files/policy-store_ok.json");
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let cedarling = Cedarling::new(BootstrapConfig {
+        application_name: "test_app".to_string(),
+        log_config: LogConfig {
+            log_type: LogTypeConfig::StdOut,
+        },
+        policy_store_config: PolicyStoreConfig {
+            source: PolicyStoreSource::Json(POLICY_STORE_RAW.to_string()),
+        },
+        jwt_config: JwtConfig::Disabled,
+    })?;
+
+    // the following tokens are expired
+
+    // access_token claims:
+    // {
+    //   "sub": "boG8dfc5MKTn37o7gsdCeyqL8LpWQtgoO41m1KZwdq0",
+    //   "code": "bf1934f6-3905-420a-8299-6b2e3ffddd6e",
+    //   "iss": "https://admin-ui-test.gluu.org",
+    //   "token_type": "Bearer",
+    //   "client_id": "5b4487c4-8db1-409d-a653-f907b8094039",
+    //   "aud": "5b4487c4-8db1-409d-a653-f907b8094039",
+    //   "acr": "basic",
+    //   "x5t#S256": "",
+    //   "scope": [
+    //     "openid",
+    //     "profile"
+    //   ],
+    //   "org_id": "some_long_id",
+    //   "auth_time": 1724830746,
+    //   "exp": 1724945978, -> Aug 29, 2024 23:39:38 GMT+0800
+    //   "iat": 1724832259, -> Aug 28, 2024 16:0419 GMT+0800
+    //   "jti": "lxTmCVRFTxOjJgvEEpozMQ",
+    //   "name": "Default Admin User",
+    //   "status": {
+    //     "status_list": {
+    //       "idx": 201,
+    //       "uri": "https://admin-ui-test.gluu.org/jans-auth/restv1/status_list"
+    //     }
+    //   }
+    // }
```

Comment:
Technically, the above is human-readable. But technically, since it's a comment, it's also unrelated to the below encoded token :thinking:

Reply:
yeah... input to Cedarling is to be expected to be in the header.claims.signature string form so it's like that in the example file :/ im not sure if that's the best way to go about it but i just followed the previous examples.